node/rpc.go

```diff
@@ -0,0 +1,133 @@
+package node
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/harmony-one/harmony/core"
+	"github.com/harmony-one/harmony/internal/hmyapi"
+)
+
+const (
+	rpcHTTPPortOffset = 10
+	rpcWSPortOffset   = 20
```

Comment:
Move all port offsets to a single file so that we won't step on each other.

Reply:
there is a ticket for this task.  https://github.com/harmony-one/harmony/issues/612, no need to block this PR.